packages/client/src/graphql/graphql.ts

```diff
@@ -0,0 +1,135 @@
+/* Generated File DO NOT EDIT. */
```

Comment:
NIT: if this is generated, should this be added to .gitignore and ensure the generation takes place before running dev, etc?

Reply:
This file is imported. I use it in the project context cause it has the Project Model in it. You don't have to have the latest up to date scheme from the server, as long as there are no breaking changes. So it also help to find out when the last update was. This file is usually you don't look at to heavly.